packages/core/package.json

```diff
@@ -1,12 +1,35 @@
 {
   "name": "@friggframework/core",
   "prettier": "@friggframework/prettier-config",
-  "version": "0.2.30",
+  "version": "1.0.0",
```

Comment:
Is this on us to do, or should this be a tag that we add to the PR into `main`, with a `major` tag so that auto/lerna does its magic?

Reply:
shrug, I think there are many ways we could do it. I don't know what we'd be gaining by trying to get lerna to do something